src/promptflow/promptflow/_core/run_tracker.py

```diff
@@ -240,11 +240,16 @@ def _ensure_serializable_value(self, val, warning_msg: Optional[str] = None):
             return ConnectionType.serialize_conn(val)
         if self.allow_generator_types and isinstance(val, GeneratorType):
             return str(val)
-        if isinstance(val, Image):
-            # Images will be persisted when persisting flow run or node run, so no need to persist it here.
-            return val
+        if isinstance(val, list):
```

Comment:
Since we have the following default logic, do we still need to check dict/list?

Reply:
No need, removed.